lib/cdc/util/geometry_test.go

```diff
@@ -15,3 +17,15 @@ func (u *UtilTestSuite) TestParseGeometryPoint() {
 		assert.Equal(u.T(), `{"type":"Feature","geometry":{"type":"Point","coordinates":[2.2945,48.8584]}}`, geoJSONString)
 	}
 }
+
+func (u *UtilTestSuite) TestGeometryWkb() {
+	{
+		geoJSONString, err := parseGeometry(map[string]interface{}{
+			"wkb":  "AQEAAAAAAAAAAADwPwAAAAAAAPA/",
+			"srid": nil,
+		})
+
+		assert.NoError(u.T(), err)
+		assert.Equal(u.T(), `{"type":"Feature","geometry":{"type":"Point","coordinates":[1,1]},"properties":null}`, geoJSONString)
```

Comment:
null exists here because the geojson library we're using to parse wkb doesn't have `omitempty`